common-lib/common/signal/base.libsonnet

```diff
@@ -182,36 +241,54 @@ local xtd = import 'github.com/jsonnet-libs/xtd/main.libsonnet';
             // keep only filteringSelector, remove any Grafana dashboard variables:
             queriesSelector: this.vars.filteringSelector[0],
           }
-          for source in sourceMaps
+          for source in this.sourceMaps
         ]
       ),
 
 
-    common::
+    common(type)::
       // override panel-wide --mixed-- datasource
       prometheusQuery.withDatasource('${%s}' % datasource)
       + g.panel.timeSeries.panelOptions.withDescription(description)
-      + g.panel.timeSeries.queryOptions.withTargets(
-        self.asTarget()
-      )
-      + self.asOverride(),
+      + (
+        if type == 'table' then {}
+        else
+          g.panel.timeSeries.queryOptions.withTargets(
+            self.asTarget()
+          )
+          + self.asOverride()
+      ),
 
     //Return as timeSeriesPanel
-    asTimeSeries(name=signalName)::
-      g.panel.timeSeries.new(name)
-      + self.common,
+    asTimeSeries(name=signalName, prototype=g.panel.timeSeries.new(name)):
+
+      // local _prototype = std.split(prototype, '.');
+      // panels[_prototype[0]][_prototype[1]][_prototype[2]].new(title='', targets=[])
```

Comment:
Leftovers from testing?

Reply:
oops, indeed it is some garbage left. Removed.